src/util/brands-url.ts

```diff
@@ -9,3 +9,5 @@ export const brandsUrl = (options: BrandsOptions): string =>
   `https://brands.home-assistant.io/${options.useFallback ? "_/" : ""}${
     options.domain
   }/${options.darkOptimized ? "dark_" : ""}${options.type}.png`;
+
+export const extractDomainFromUrl = (url: string) => url.split("/")[4];
```

Comment:
It doesn't extract this from any URL, just brand URLs

```suggestion
export const extractDomainFromBrandUrl = (url: string) => url.split("/")[4];
```

Reply:
It is in "brands-url.ts", but ok.

Reply:
Yeah but the auto-complete in VS Code includes the constants of all files.